glslc/test/include.py

```diff
@@ -142,21 +151,23 @@ class VerifyNestedIncludeAmongSiblings(expect.StdoutMatch):
     """Tests #include inside #included sibling files."""
 
     environment = Directory('.', [
-        File('a.vert', '#include "b"\ncontent a\n'),
+        File('a.vert', '#version 140\n#include "b"\ncontent a\n'),
         File('b', 'content b\n#include "c"\n'),
         File('c', 'content c\n')])
 
     glslc_args = ['-E', 'a.vert']
 
     expected_stdout = \
-"""#extension GL_GOOGLE_include_directive : enable
+"""#version 140
+#extension GL_GOOGLE_include_directive : enable
 #line 0 "a.vert"
```

Comment:
I was referring to the fact that a line was inserted into `a.vert` so all `#line`s in the output should have shifted, yet not all did.  But I withdraw my comment on the basis that there's no spec for this, and we can declare `#version` special.